code/modules/admin/tabs/round_tab.dm

```diff
@@ -165,12 +165,13 @@
 	set name = "Start Round"
 	set desc = "Start the round RIGHT NOW"
 	set category = "Server.Round"
-
-	if (!SSticker)
-		alert("Unable to start the game as it is not set up.")
-		return
 	if (alert("Are you sure you want to start the round early?",,"Yes","No") != "Yes")
 		return
+	if (SSticker.current_state == GAME_STATE_STARTUP)
+		alert("Game is setting up and will launch as soon as it is ready.")
```

Comment:
given alert sleeps until it's pressed, do we actually want this, given you can't cancel *out* of an alert anyway?

Reply:
Modified